src/platform/apollolake/include/platform/platform.h

```diff
@@ -104,7 +104,7 @@ struct sof;
  * the interval of reschedule DMA trace copying in special case like half
  * fullness of local DMA trace buffer
  */
-#define DMA_TRACE_RESCHEDULE_TIME	5000
+#define DMA_TRACE_RESCHEDULE_TIME	5
```

Comment:
This means trace is scheduled every 5us. Please schedule trace every 5ms unless buffer is > 1/2 full then schedule next flush in 500us.

Reply:
No, it is scheduled every 500ms,  unless buffer is > 1/2 full then schedule next flush in 5us.

Reply:
I discussed with xiuli and keyon,  it is not reasonable the reschedule time is  500us. The trace buffer should be rescheduled ASAP because it is possible overflow for a while.

Reply:
ok, go it now. Probably best to include this level of detail in next PR message as it wasn't clear.